src/modules/75-cd/components/PipelineSteps/Common/Terraform/Editview/TerraformArtifactoryForm.tsx

```diff
@@ -238,9 +257,14 @@ export const TFArtifactoryForm: React.FC<StepProps<any> & TFArtifactoryProps> =
                 <div className={cx(stepCss.md)}>
                   <MultiTypeFieldSelector
                     name={tfArtifactoryFormInputNames(isConfig).artifactPaths}
-                    label={getString(isConfig ? 'pipeline.artifactPathLabel' : 'cd.artifactPaths')}
                     style={{ width: 370 }}
                     allowedTypes={allowableTypes.filter(item => item !== MultiTypeInputType.EXPRESSION)}
+                    label={
+                      <Text flex={{ inline: true }}>
+                        {getString(isConfig ? 'pipeline.artifactPathLabel' : 'cd.artifactPaths')}
+                        <HarnessDocTooltip useStandAlone={true} tooltipId="artifactory_file_path" />
+                      </Text>
```

Comment:
You can actually simplify this and drop the `HarnessDocTooltip`
```suggestion
                      <Text
                        flex={{ inline: true }}
                        tooltipProps={{ tooltipId: 'artifactory_file_path' }}
                      >
                        {getString(isConfig ? 'pipeline.artifactPathLabel' : 'cd.artifactPaths')}
                      </Text>
```